### PR TITLE
W-012

### PR DESCRIPTION
**Pull request by WOJTASK1**
This is the latest version of navbar component. I kindly ask for a PR.

### CODE REVIEW

File: frontend/src/components/navbar/Navbar.css
Comment (MartynaSzymanskaGitHub) [inline frontend/src/components/navbar/Navbar.css:1]: Change colors variables from GlobalColors.css

import @import "../../GlobalColors.css";
and change colors to : example : var(--color-button)

**Review by MartynaSzymanskaGitHub — CHANGES_REQUESTED** [review]
needs changing described below

File: frontend/src/components/navbar/Navbar.css
Comment (WOJTASK1) [reply frontend/src/components/navbar/Navbar.css:1]: Thanks for your advice.

File: frontend/src/components/navbar/Navbar.jsx
[the remark this answers is not in the document]
Comment (WOJTASK1) [reply frontend/src/components/navbar/Navbar.jsx:1]: We have already talked about this issue (too short navbar). My recommendation is to postpone fixing this bug in order to focus on more challenging tasks.

**Pull request closed without merging**